### ISSUE TITLE
Revisit date picker

### ISSUE THREAD

**Issue by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1257]
The date picker is still annoying to work with - typing dates often does not work because the calendar widget overlays the input field (Chrome, full screen), and selecting years is cumbersome. Arguably, we want widgets which are convenient for historical dates, while most calendar widgets are designed for picking dates close to the current date.

**Comment by lukavdplas** [comment]
agreed, i don't think having a calendar widget is really appropriate here.

As a side note, modern browsers generally have a built-in calendar widget with `<input type="date">`. If that improves compatibility, it's a minimal change.

I would welcome a redesign!

One issue that I have with the calendar is that you often use a rough filter on dates, where you only care about the exact date (e.g "after 1980"), but now you always have to select a specific day anyway.
